model/client4.go

```diff
@@ -57,6 +57,10 @@ func (c *Client4) GetUsersRoute() string {
 	return fmt.Sprintf("/users")
 }
 
+func (c *Client4) GetAutocompleteUsersRoute() string {
+	return fmt.Sprintf("/users/autocomplete/")
+}
```

Comment:
I'd say this is a bit overkill, just use `GetUsersRoute()` in your driver function and append '/autocomplete'